CHANGELOG.md

```diff
@@ -2,10 +2,20 @@
 
 ## 0.9.28
 
+### New features
+
+* [#3862](https://github.com/buefy/buefy/issues/3862) Expose `isActiveDetailRow` method in the default scoped slot props of table column (thanks @OrestHk)
+* [#3863](https://github.com/buefy/buefy/issues/3863) Add `active` event emitted when dropdown turns active/inactive (thanks @daniil4udo)
+
 ### Fixes
 
 * Fix [#3961](https://github.com/buefy/buefy/issues/3961) Dropdown with "hover" trigger incorrectly rendered on mobile devices
-* Fix [#3963](https://github.com/buefy/buefy/issues/3963) no dropdown showed up if the data array was empty and only the footer slot was registered (thanks @GMartigny)
+* Fix [#3963](https://github.com/buefy/buefy/issues/3963) No dropdown showed up if the data array was empty and only the footer slot was registered (thanks @GMartigny)
+* Fix [#3971](https://github.com/buefy/buefy/issues/3971) Problems with table headers and dates due to 0.9.25 → 0.9.27 changes
+* Fix [#3971](https://github.com/buefy/buefy/issues/3971) Percentage width on table columns doubly applied
+* Fix [#3991](https://github.com/buefy/buefy/pull/3991) Potential call after unmount in `NumberInput` (thanks @Arooba-git)
+* Fix [#3991](https://github.com/buefy/buefy/pull/3991) Potential call after unmount in `Sidebar` (thanks @Arooba-git)
+* Fix [#3991](https://github.com/buefy/buefy/pull/3991) Potential listener leaks in `SliderThumb` (thanks @Arooba-git)
```

Comment:
@wesdevpro Sorry for confusion, but I appended fixes made in #3991 to CHANGELOG.

Reply:
Sounds good!